android/KMEA/app/src/main/java/com/tavultesoft/kmea/KeyboardPickerActivity.java

```diff
@@ -636,26 +537,7 @@ protected static void notifyKeyboardsUpdate(Context context) {
     storage.keyboards.setNotifyOnChange(false);
     storage.keyboards.clear();
 
-    List<HashMap<String, String>> mapList = getKeyboardsList(context);
-    List<Keyboard> kbdList = new ArrayList<>(mapList.size());
-    for(HashMap<String, String> kbdMap: mapList) {
-      boolean isNewKeyboard = kbdMap.containsKey(KeyboardPickerActivity.KMKEY_INTERNAL_NEW_KEYBOARD) &&
-        kbdMap.get(KeyboardPickerActivity.KMKEY_INTERNAL_NEW_KEYBOARD).equals(KeyboardPickerActivity.KMKEY_INTERNAL_NEW_KEYBOARD);
-
-      Keyboard k = new Keyboard(
-        kbdMap.get(KMManager.KMKey_PackageID),
-        kbdMap.get(KMManager.KMKey_KeyboardID),
-        kbdMap.get(KMManager.KMKey_KeyboardName),
-        kbdMap.get(KMManager.KMKey_LanguageID),
-        kbdMap.get(KMManager.KMKey_LanguageName),
-        MapCompat.getOrDefault(kbdMap, KMManager.KMKey_Version, "1.0"),
-        MapCompat.getOrDefault(kbdMap, KMManager.KMKey_CustomHelpLink, ""),
-        isNewKeyboard,
-        MapCompat.getOrDefault(kbdMap, KMManager.KMKey_Font, null),
-        MapCompat.getOrDefault(kbdMap, KMManager.KMKey_OskFont, null)
-      );
-      kbdList.add(k);
-    }
+    List<Keyboard> kbdList = KeyboardController.getInstance().get();
```

Comment:
A bit sad that this method (`notifyKeyboardsUpdate`) and the next (`notifyLexicalModelsUpdate`) are still in-place within `KeyboardPickerActivity`, rather than part of the new `KeyboardController`, but I realize that may be better left to a followup PR.  (`KeyboardController` didn't exist when these methods were written.)